src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -168,9 +168,11 @@ def safe_to_destroy(self, svc_id):
             svc_id = [svc_id]
         svc_id = list(map(str, svc_id))
         try:
-            CephService.send_command(
+            result = CephService.send_command(
                 'mon', 'osd safe-to-destroy', ids=svc_id, target=('mgr', ''))
-            return {'safe-to-destroy': True}
+            return {'safe-to-destroy': set(result['safe_to_destroy']) ==
+                    set(map(lambda id: int(id), svc_id)),
```

Comment:
`set(map(int, svc_id))` suffices, but as this is blocking QA I'll leave it as it is for now